Typo in incrementer name leads to wrong exception

Typo in incrementer name:

```
semver {
    snapshotSuffix = "SNAPSHOT"
    dirtyMarker = "dirty"
    initialVersion = "0.1.0"
    branches {
        branch {
            regex = ".+"
            incrementer = "NO_VERSION1_INCREMENTER"
            formatter = Transformer { info:Info -> "${info.version.major}.${info.version.minor}.${info.version.patch}" }
        }
    }
}
```
instead of
```
semver {
    snapshotSuffix = "SNAPSHOT"
    dirtyMarker = "dirty"
    initialVersion = "0.1.0"
    branches {
        branch {
            regex = ".+"
            incrementer = "NO_VERSION_INCREMENTER"
            formatter = Transformer { info:Info -> "${info.version.major}.${info.version.minor}.${info.version.patch}" }
        }
    }
}
```
leads to:

```
org.gradle.internal.exceptions.LocationAwareException: Failed to notify dependency resolution listener.
	at org.gradle.initialization.exception.DefaultExceptionAnalyser.transform(DefaultExceptionAnalyser.java:99)
	at org.gradle.initialization.exception.DefaultExceptionAnalyser.collectFailures(DefaultExceptionAnalyser.java:65)
	at org.gradle.initialization.exception.MultipleBuildFailuresExceptionAnalyser.transform(MultipleBuildFailuresExceptionAnalyser.java:39)
	at org.gradle.initialization.exception.StackTraceSanitizingExceptionAnalyser.transform(StackTraceSanitizingExceptionAnalyser.java:29)
	at org.gradle.initialization.DefaultGradleLauncher.finishBuild(DefaultGradleLauncher.java:194)
	at org.gradle.initialization.DefaultGradleLauncher.finishBuild(DefaultGradleLauncher.java:141)
	at org.gradle.internal.invocation.GradleBuildController$3.create(GradleBuildController.java:83)
	at org.gradle.internal.invocation.GradleBuildController$3.create(GradleBuildController.java:75)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:183)
	at org.gradle.internal.work.StopShieldingWorkerLeaseService.withLocks(StopShieldingWorkerLeaseService.java:40)
	at org.gradle.internal.invocation.GradleBuildController.doBuild(GradleBuildController.java:75)
	at org.gradle.internal.invocation.GradleBuildController.configure(GradleBuildController.java:64)
	at org.gradle.tooling.internal.provider.runner.ClientProvidedBuildActionRunner.run(ClientProvidedBuildActionRunner.java:57)
	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
	at org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:58)
	at org.gradle.tooling.internal.provider.ValidatingBuildActionRunner.run(ValidatingBuildActionRunner.java:32)
	at org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:39)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:51)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:102)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner.run(RunAsBuildOperationBuildActionRunner.java:45)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter$1.transform(InProcessBuildActionExecuter.java:49)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter$1.transform(InProcessBuildActionExecuter.java:46)
	at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:78)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:46)
	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:31)
	at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:42)
	at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:28)
	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:78)
	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:52)
	at org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:59)
	at org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:36)
	at org.gradle.tooling.internal.provider.SessionScopeBuildActionExecuter.execute(SessionScopeBuildActionExecuter.java:68)
	at org.gradle.tooling.internal.provider.SessionScopeBuildActionExecuter.execute(SessionScopeBuildActionExecuter.java:38)
	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:37)
	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:26)
	at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:43)
	at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:29)
	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:60)
	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:32)
	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:55)
	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:41)
	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:48)
	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:32)
	at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:67)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:26)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:34)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:74)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:72)
	at org.gradle.util.Swapper.swap(Swapper.java:38)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:72)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:62)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:81)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:50)
	at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:295)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: org.gradle.internal.event.ListenerNotificationException: Failed to notify dependency resolution listener.
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:86)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:324)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:234)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:140)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy35.beforeResolve(Unknown Source)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.performPreResolveActions(DefaultConfiguration.java:638)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.access$800(DefaultConfiguration.java:138)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$7.run(DefaultConfiguration.java:583)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.resolveGraphIfRequired(DefaultConfiguration.java:576)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.access$600(DefaultConfiguration.java:138)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$6.run(DefaultConfiguration.java:556)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$SafeExclusiveLockImpl.withLock(DefaultProjectStateRegistry.java:244)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.resolveExclusively(DefaultConfiguration.java:552)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.resolveToStateOrLater(DefaultConfiguration.java:547)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.access$2200(DefaultConfiguration.java:138)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$ConfigurationFileCollection.getSelectedArtifacts(DefaultConfiguration.java:1178)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$ConfigurationFileCollection.access$3600(DefaultConfiguration.java:1113)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$ConfigurationArtifactCollection.ensureResolved(DefaultConfiguration.java:1628)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$ConfigurationArtifactCollection.iterator(DefaultConfiguration.java:1612)
	at org.gradle.plugins.ide.internal.resolver.IdeDependencySet$IdeDependencyResult.resolvePlusConfigurations(IdeDependencySet.java:98)
	at org.gradle.plugins.ide.internal.resolver.IdeDependencySet$IdeDependencyResult.visit(IdeDependencySet.java:88)
	at org.gradle.plugins.ide.internal.resolver.IdeDependencySet.visit(IdeDependencySet.java:71)
	at org.gradle.plugins.ide.idea.model.internal.IdeaDependenciesProvider$1.create(IdeaDependenciesProvider.java:113)
	at org.gradle.plugins.ide.idea.model.internal.IdeaDependenciesProvider$1.create(IdeaDependenciesProvider.java:109)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.withMutableState(DefaultProjectStateRegistry.java:192)
	at org.gradle.plugins.ide.idea.model.internal.IdeaDependenciesProvider.visitDependencies(IdeaDependenciesProvider.java:109)
	at org.gradle.plugins.ide.idea.model.internal.IdeaDependenciesProvider.getDependencies(IdeaDependenciesProvider.java:93)
	at org.gradle.plugins.ide.idea.model.internal.IdeaDependenciesProvider.provide(IdeaDependenciesProvider.java:69)
	at org.gradle.plugins.ide.idea.model.IdeaModule.resolveDependencies(IdeaModule.java:575)
	at org.gradle.plugins.ide.internal.tooling.IdeaModelBuilder.buildDependencies(IdeaModelBuilder.java:119)
	at org.gradle.plugins.ide.internal.tooling.IdeaModelBuilder.createModule(IdeaModelBuilder.java:178)
	at org.gradle.plugins.ide.internal.tooling.IdeaModelBuilder.build(IdeaModelBuilder.java:107)
	at org.gradle.plugins.ide.internal.tooling.IdeaModelBuilder.buildAll(IdeaModelBuilder.java:76)
	at org.gradle.plugins.ide.internal.tooling.IdeaModelBuilder.buildAll(IdeaModelBuilder.java:57)
	at org.gradle.tooling.provider.model.internal.DefaultToolingModelBuilderRegistry$BuildOperationWrappingToolingModelBuilder$1$1.create(DefaultToolingModelBuilderRegistry.java:102)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry.withLenientState(DefaultProjectStateRegistry.java:132)
	at org.gradle.tooling.provider.model.internal.DefaultToolingModelBuilderRegistry$BuildOperationWrappingToolingModelBuilder$1.call(DefaultToolingModelBuilderRegistry.java:98)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:102)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
	at org.gradle.tooling.provider.model.internal.DefaultToolingModelBuilderRegistry$BuildOperationWrappingToolingModelBuilder.buildAll(DefaultToolingModelBuilderRegistry.java:95)
	at org.gradle.tooling.internal.provider.runner.DefaultBuildController.getModel(DefaultBuildController.java:79)
	at org.gradle.tooling.internal.consumer.connection.InternalBuildActionAdapter$2.getModel(InternalBuildActionAdapter.java:77)
	at org.gradle.tooling.internal.consumer.connection.BuildControllerAdapter.getModel(BuildControllerAdapter.java:62)
	at org.gradle.tooling.internal.consumer.connection.AbstractBuildController.getModel(AbstractBuildController.java:40)
	at org.gradle.tooling.internal.consumer.connection.AbstractBuildController.getModel(AbstractBuildController.java:28)
	at org.jetbrains.plugins.gradle.model.ProjectImportAction.execute(ProjectImportAction.java:74)
	at org.jetbrains.plugins.gradle.model.ProjectImportAction.execute(ProjectImportAction.java:41)
	at org.gradle.tooling.internal.consumer.connection.InternalBuildActionAdapter.execute(InternalBuildActionAdapter.java:80)
	at org.gradle.tooling.internal.provider.runner.ClientProvidedBuildActionRunner$ResultBuildingListener.buildResult(ClientProvidedBuildActionRunner.java:114)
	at org.gradle.tooling.internal.provider.runner.ClientProvidedBuildActionRunner$ResultBuildingListener.buildFinished(ClientProvidedBuildActionRunner.java:106)
	at jdk.internal.reflect.GeneratedMethodAccessor46.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.event.DefaultListenerManager$ListenerDetails.dispatch(DefaultListenerManager.java:376)
	at org.gradle.internal.event.DefaultListenerManager$ListenerDetails.dispatch(DefaultListenerManager.java:358)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:58)
	at org.gradle.internal.event.DefaultListenerManager$EventBroadcast$ListenerDispatch.dispatch(DefaultListenerManager.java:346)
	at org.gradle.internal.event.DefaultListenerManager$EventBroadcast$ListenerDispatch.dispatch(DefaultListenerManager.java:333)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:42)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:230)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:149)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:58)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:324)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:234)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:140)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy14.buildFinished(Unknown Source)
	at org.gradle.initialization.DefaultGradleLauncher.finishBuild(DefaultGradleLauncher.java:179)
	... 73 more
Caused by: org.gradle.api.GradleException: Failed to resolve imported Maven boms: The last tag is not a semantic version.
	at io.spring.gradle.dependencymanagement.internal.DependencyManagement.resolveIfNecessary(DependencyManagement.java:157)
	at io.spring.gradle.dependencymanagement.internal.DependencyManagement.getManagedVersions(DependencyManagement.java:121)
	at io.spring.gradle.dependencymanagement.internal.DependencyManagementContainer.getManagedVersionsForConfiguration(DependencyManagementContainer.java:202)
	at io.spring.gradle.dependencymanagement.internal.DependencyManagementContainer.getManagedVersionsForConfiguration(DependencyManagementContainer.java:187)
	at io.spring.gradle.dependencymanagement.internal.DependencyManagementApplier$1.execute(DependencyManagementApplier.java:76)
	at io.spring.gradle.dependencymanagement.internal.DependencyManagementApplier$1.execute(DependencyManagementApplier.java:72)
	at org.gradle.configuration.internal.DefaultUserCodeApplicationContext$2.execute(DefaultUserCodeApplicationContext.java:77)
	at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:91)
	at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:80)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:42)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:230)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:149)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:58)
	... 156 more
Caused by: org.gradle.api.artifacts.ResolveException: Could not resolve all dependencies for configuration ':detachedConfiguration11'.
	at org.gradle.api.internal.artifacts.ivyservice.ErrorHandlingConfigurationResolver.wrapException(ErrorHandlingConfigurationResolver.java:103)
	at org.gradle.api.internal.artifacts.ivyservice.ErrorHandlingConfigurationResolver.resolveGraph(ErrorHandlingConfigurationResolver.java:75)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$7.run(DefaultConfiguration.java:585)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.resolveGraphIfRequired(DefaultConfiguration.java:576)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.access$600(DefaultConfiguration.java:138)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration$6.run(DefaultConfiguration.java:556)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$SafeExclusiveLockImpl.withLock(DefaultProjectStateRegistry.java:244)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.resolveExclusively(DefaultConfiguration.java:552)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.resolveToStateOrLater(DefaultConfiguration.java:547)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.getResolvedConfiguration(DefaultConfiguration.java:529)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration_Decorated.getResolvedConfiguration(Unknown Source)
	at io.spring.gradle.dependencymanagement.internal.maven.MavenPomResolver.resolvePoms(MavenPomResolver.java:83)
	at io.spring.gradle.dependencymanagement.internal.DependencyManagement.resolve(DependencyManagement.java:183)
	at io.spring.gradle.dependencymanagement.internal.DependencyManagement.resolveIfNecessary(DependencyManagement.java:153)
	... 168 more
Caused by: org.gradle.api.GradleException: The last tag is not a semantic version.
	at io.wusa.VersionService.getVersion(VersionService.kt:31)
	at io.wusa.Info.getVersion(Info.kt:56)
	at io.wusa.formatter.SemanticVersionFormatter$Companion.format(SemanticVersionFormatter.kt:20)
	at io.wusa.Info.toString(Info.kt:61)
	at org.gradle.api.internal.artifacts.ProjectBackedModule.getVersion(ProjectBackedModule.java:38)
	at org.gradle.api.internal.artifacts.ivyservice.moduleconverter.DefaultRootComponentMetadataBuilder.buildRootComponentMetadata(DefaultRootComponentMetadataBuilder.java:82)
	at org.gradle.api.internal.artifacts.ivyservice.moduleconverter.DefaultRootComponentMetadataBuilder.toRootComponentMetaData(DefaultRootComponentMetadataBuilder.java:75)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.toRootComponentMetaData(DefaultConfiguration.java:992)
	at org.gradle.api.internal.artifacts.ivyservice.resolveengine.DefaultArtifactDependencyResolver$DefaultResolveContextToComponentResolver.resolve(DefaultArtifactDependencyResolver.java:182)
	at org.gradle.api.internal.artifacts.ivyservice.resolveengine.graph.builder.DependencyGraphBuilder.resolve(DependencyGraphBuilder.java:135)
	at org.gradle.api.internal.artifacts.ivyservice.resolveengine.DefaultArtifactDependencyResolver.resolve(DefaultArtifactDependencyResolver.java:122)
	at org.gradle.api.internal.artifacts.ivyservice.DefaultConfigurationResolver.resolveGraph(DefaultConfigurationResolver.java:171)
	at org.gradle.api.internal.artifacts.ivyservice.ShortCircuitEmptyConfigurationResolver.resolveGraph(ShortCircuitEmptyConfigurationResolver.java:86)
	at org.gradle.api.internal.artifacts.ivyservice.ErrorHandlingConfigurationResolver.resolveGraph(ErrorHandlingConfigurationResolver.java:73)
	... 187 more
```

This should return a custom exception e.g., NoSuchIncrementerException

--- a/src/main/kotlin/io/wusa/GitService.kt
+++ b/src/main/kotlin/io/wusa/GitService.kt
@@ -44,7 +44,7 @@ class GitService {
             }
         }
 
-        @Throws(NoCurrentTagFoundException::class)
+        @Throws(NoLastTagFoundException::class)
         fun lastTag(project: Project): String {
             return try {
                 GitCommandRunner.execute(project.projectDir, arrayOf("describe", "--dirty", "--abbrev=7"))

--- a/src/main/kotlin/io/wusa/IVersionFactory.kt
+++ b/src/main/kotlin/io/wusa/IVersionFactory.kt
@@ -1,6 +1,9 @@
 package io.wusa
 
+import io.wusa.exception.NoValidSemverTagFoundException
+
 interface IVersionFactory {
 
+    @Throws(NoValidSemverTagFoundException::class)
     fun createFromString(describe: String): Version
 }

--- a/src/main/kotlin/io/wusa/SemanticVersionFactory.kt
+++ b/src/main/kotlin/io/wusa/SemanticVersionFactory.kt
@@ -1,26 +1,31 @@
 package io.wusa
 
+import io.wusa.exception.NoValidSemverTagFoundException
 import org.slf4j.LoggerFactory
 
 class SemanticVersionFactory : IVersionFactory {
     private val LOG = LoggerFactory.getLogger(SemanticVersionFactory::class.java)
 
     override fun createFromString(describe: String): Version {
-        val suffixRegex = """(?:-(?<count>[0-9]+)(?:-g(?<sha>[0-9a-f]{1,7}))(?<dirty>-dirty)?)$""".toRegex()
-        val suffix = suffixRegex.find(describe)
-                ?.destructured
-                ?.let { (count, sha, dirty) ->
-                    Suffix(count.toInt(), sha, dirty.isNotEmpty())
-                }
+        try {
+            val suffixRegex = """(?:-(?<count>[0-9]+)(?:-g(?<sha>[0-9a-f]{1,7}))(?<dirty>-dirty)?)$""".toRegex()
+            val suffix = suffixRegex.find(describe)
+                    ?.destructured
+                    ?.let { (count, sha, dirty) ->
+                        Suffix(count.toInt(), sha, dirty.isNotEmpty())
+                    }
 
-        var version = describe
-        if (suffix != null) {
-            version = suffixRegex.replace(describe, "")
+            var version = describe
+            if (suffix != null) {
+                version = suffixRegex.replace(describe, "")
+            }
+
+            val parsedVersion = parseVersion(version)
+            parsedVersion.suffix = suffix
+            return parsedVersion
+        } catch (ex: IllegalArgumentException) {
+            throw NoValidSemverTagFoundException("The last tag is not a semantic version.")
         }
-
-        val parsedVersion = parseVersion(version)
-        parsedVersion.suffix = suffix
-        return parsedVersion
     }
 
     private fun parseVersion(version: String): Version {

--- a/src/main/kotlin/io/wusa/VersionService.kt
+++ b/src/main/kotlin/io/wusa/VersionService.kt
@@ -1,8 +1,6 @@
 package io.wusa
 
-import io.wusa.exception.NoCurrentCommitFoundException
-import io.wusa.exception.NoCurrentTagFoundException
-import io.wusa.exception.NoLastTagFoundException
+import io.wusa.exception.*
 import io.wusa.extension.SemverGitPluginExtension
 import io.wusa.incrementer.VersionIncrementer
 import org.gradle.api.GradleException
@@ -27,8 +25,10 @@ class VersionService(private var project: Project) {
                 } ?: run {
                     return VersionIncrementer.getVersionIncrementerByName(SemverGitPluginExtension.DEFAULT_INCREMENTER).increment(version)
                 }
-            } catch (ex: IllegalArgumentException) {
-                throw GradleException("The last tag is not a semantic version.")
+            } catch (ex: NoValidSemverTagFoundException) {
+                throw GradleException(ex.localizedMessage)
+            } catch (ex: NoIncrementerFoundException) {
+                throw GradleException(ex.localizedMessage)
             } catch (ex: NoLastTagFoundException) {
                 return try {
                     val sha = GitService.currentCommit(project, true)

--- a/src/main/kotlin/io/wusa/exception/NoIncrementerFoundException.kt
+++ b/src/main/kotlin/io/wusa/exception/NoIncrementerFoundException.kt
@@ -1,0 +1,7 @@
+package io.wusa.exception
+
+class NoIncrementerFoundException : Exception {
+    constructor(message: String, ex: Throwable?) : super(message, ex) {}
+    constructor(message: String) : super(message) {}
+    constructor(ex: Throwable) : super(ex) {}
+}

--- a/src/main/kotlin/io/wusa/exception/NoValidSemverTagFoundException.kt
+++ b/src/main/kotlin/io/wusa/exception/NoValidSemverTagFoundException.kt
@@ -1,0 +1,7 @@
+package io.wusa.exception
+
+class NoValidSemverTagFoundException : Exception {
+    constructor(message: String, ex: Throwable?) : super(message, ex) {}
+    constructor(message: String) : super(message) {}
+    constructor(ex: Throwable) : super(ex) {}
+}

--- a/src/main/kotlin/io/wusa/incrementer/VersionIncrementer.kt
+++ b/src/main/kotlin/io/wusa/incrementer/VersionIncrementer.kt
@@ -1,6 +1,8 @@
 package io.wusa.incrementer
 
 import io.wusa.Version
+import io.wusa.exception.NoIncrementerFoundException
+import java.lang.IllegalArgumentException
 
 enum class VersionIncrementer: IIncrementer {
     NO_VERSION_INCREMENTER {
@@ -25,6 +27,12 @@ enum class VersionIncrementer: IIncrementer {
     };
 
     companion object {
-        fun getVersionIncrementerByName(name: String) = valueOf(name.toUpperCase())
+        fun getVersionIncrementerByName(name: String): VersionIncrementer {
+            try {
+                return valueOf(name.toUpperCase())
+            } catch (ex: IllegalArgumentException) {
+                throw NoIncrementerFoundException("The in the config specified incrementer was not found.")
+            }
+        }
     }
 }

--- a/src/test/kotlin/io/wusa/SemanticVersionFactoryTest.kt
+++ b/src/test/kotlin/io/wusa/SemanticVersionFactoryTest.kt
@@ -1,12 +1,12 @@
 package io.wusa
 
 import io.mockk.mockkObject
+import io.wusa.exception.NoValidSemverTagFoundException
 import org.gradle.api.Project
 import org.gradle.testfixtures.ProjectBuilder
 import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
-import java.lang.IllegalArgumentException
 
 class SemanticVersionFactoryTest {
 
@@ -73,136 +73,136 @@ class SemanticVersionFactoryTest {
         Assertions.assertEquals(semanticVersionFactory.createFromString("1.0.0-5"), Version(1, 0, 0, "5", "", null))
         Assertions.assertEquals(semanticVersionFactory.createFromString("1.0.0-5-g"), Version(1, 0, 0, "5-g", "", null))
 
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2.3-0123")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2.3-0123.0123")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.1.2+.123")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("+invalid")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("-invalid")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("-invalid+invalid")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("-invalid.01")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("alpha")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("alpha.beta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("alpha.beta.1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("alpha.1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("alpha+beta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("alpha_beta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("alpha..")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("beta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha_beta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("-")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha..")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha..1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha...1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha....1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha.....1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha......1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-alpha.......1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("01.1.1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.01.1")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.1.01")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2.3.DEV")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2-SNAPSHOT")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2.31.2.3----RC-SNAPSHOT.12.09.1--..12+788")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.2-RC-SNAPSHOT")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("-1.0.3-gamma+b7718")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("+justmeta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("9.8.7+meta+meta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("9.8.7-whatever+meta+meta")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("9999999.9999999.9999999----RC-SNAPSHOT.12.09.1--------------------------------..12")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.1.1 ")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("111")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("a.b.c")
         }
-        Assertions.assertThrows(IllegalArgumentException::class.java) {
+        Assertions.assertThrows(NoValidSemverTagFoundException::class.java) {
             semanticVersionFactory.createFromString("1.0.0-")
         }
     }

--- a/src/test/kotlin/io/wusa/SemverGitPluginKotlinFunctionalTest.kt
+++ b/src/test/kotlin/io/wusa/SemverGitPluginKotlinFunctionalTest.kt
@@ -1,6 +1,7 @@
 package io.wusa
 
 import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.UnexpectedBuildFailure
 import org.junit.jupiter.api.*
 import java.io.File
 
@@ -122,6 +123,38 @@ class SemverGitPluginKotlinFunctionalTest : FunctionalBaseTest() {
                 .build()
         println(result.output)
         Assertions.assertTrue(result.output.contains("Version: 0.1.0-SNAPSHOT"))
+    }
+
+    @Test
+    fun `version wrong incrementer`() {
+        val testProjectDirectory = createTempDir()
+        val buildFile = File(testProjectDirectory, "build.gradle.kts")
+        buildFile.writeText("""
+            import io.wusa.Info
+
+            plugins {
+                id("io.wusa.semver-git-plugin")
+            }
+
+            semver {
+                branches {
+                    branch {
+                        regex = ".*"
+                        incrementer = "THIS_IS_NO_INCREMENTer"
+                        formatter = Transformer<Any, Info>{ "${'$'}{info.version.major}.${'$'}{info.version.minor}.${'$'}{info.version.patch}+build.${'$'}{info.count}.sha.${'$'}{info.shortCommit}" }
+                    }
+                }
+            }
+        """)
+        val git = initializeGitWithBranch(testProjectDirectory, "0.0.1")
+        git.commit().setMessage("").call()
+        Assertions.assertThrows(UnexpectedBuildFailure::class.java) {
+            gradleRunner
+                    .withProjectDir(testProjectDirectory)
+                    .withArguments("showVersion")
+                    .withPluginClasspath()
+                    .build()
+        }
     }
 
     @Test

--- a/src/test/kotlin/io/wusa/VersionIncrementerTest.kt
+++ b/src/test/kotlin/io/wusa/VersionIncrementerTest.kt
@@ -1,0 +1,36 @@
+package io.wusa
+
+import io.wusa.exception.NoIncrementerFoundException
+import io.wusa.incrementer.VersionIncrementer
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class VersionIncrementerTest {
+
+    @Test
+    fun `no increment`() {
+        Assertions.assertEquals(VersionIncrementer.getVersionIncrementerByName("NO_VERSION_INCREMENTER").increment(Version(0, 0, 0, "", "", null)), Version(0, 0, 0, "", "", null))
+    }
+
+    @Test
+    fun `patch increment`() {
+        Assertions.assertEquals(VersionIncrementer.getVersionIncrementerByName("PATCH_INCREMENTER").increment(Version(0, 0, 0, "", "", null)), Version(0, 0, 1, "", "", null))
+    }
+
+    @Test
+    fun `minor increment`() {
+        Assertions.assertEquals(VersionIncrementer.getVersionIncrementerByName("MINOR_INCREMENTER").increment(Version(0, 0, 0, "", "", null)), Version(0, 1, 0, "", "", null))
+    }
+
+    @Test
+    fun `major increment`() {
+        Assertions.assertEquals(VersionIncrementer.getVersionIncrementerByName("MAJOR_INCREMENTER").increment(Version(0, 0, 0, "", "", null)), Version(1, 0, 0, "", "", null))
+    }
+
+    @Test
+    fun `wrong incrementer`() {
+        Assertions.assertThrows(NoIncrementerFoundException::class.java) {
+            VersionIncrementer.getVersionIncrementerByName("WRONG_INCREMENTER").increment(Version(0, 0, 0, "", "", null))
+        }
+    }
+}

